website_odoo_debranding/templates/disable_odoo.xml

```diff
@@ -1,9 +1,8 @@
-<?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
     <!-- Remove "Create a free website with Odoo" -->
     <template
         id="layout_footer_copyright"
-        inherit_id="website.brand_promotion"
+        inherit_id="web.brand_promotion"
```

Comment:
Why did you change this? It seems not necessary to me.

Reply:
you are right! not necessary. happens, as this promotion message comes from web. my bad...